File: test-db.js
import { Database } from 'sqlite-async';

async function check() {
    try {
        const db = await Database.open('./database.sqlite');
        const rows = await db.all("SELECT qmqa_id, request_status FROM QMQA WHERE qmqa_id='6eea41f3-8b23-414f-9cf0-b6c3c97ff5c0'");
        console.log(rows);
    } catch(e){
        console.error(e);
    }
}
check();

File: test-db.py
import sqlite3

try:
    conn = sqlite3.connect('database.sqlite')
    c = conn.cursor()
    c.execute("SELECT name FROM sqlite_master WHERE type='table';")
    tables = c.fetchall()
    print(f"Tables: {tables}")
    
    # Just in case the table name is different
    for t in tables:
        if 'qmqa' in t[0].lower():
            c.execute(f"PRAGMA table_info({t[0]});")
            print(f"Schema for {t[0]}: {c.fetchall()}")
except Exception as e:
    print(e)
finally:
    if 'conn' in locals():
        conn.close()
